src/Illuminate/Auth/AuthManager.php

```diff
@@ -80,23 +80,19 @@ public function guard($name = null)
      */
     protected function resolve($name)
     {
-        $config = $this->getConfig($name);
-
-        if (is_null($config)) {
+        if (is_null($config = $this->getConfig($name))) {
```

Comment:
I'm meaning I personally avoid them, and I think Taylor generally does too. There's no real benefit to doing it here, so I'd suggest we don't. :)